whois-api/src/test/java/net/ripe/db/whois/api/rest/TrustedQueryTestIntegration.java

```diff
@@ -86,18 +86,12 @@ public void setup() {
     // inverse lookup on sponsoring org attribute
 
     @Test
-    public void inverse_lookup_sponsoring_org_from_untrusted_range_returns_empty() {
+    public void inverse_lookup_sponsoring_org_from_untrusted_range_succeeds() {
```

Comment:
Move this test to a different Query Integration test class (alongside "org:" maybe?), as "trusted" or not is no longer relevant

Reply:
I have already WhoisSearchServiceTestIntegration.search_inverse_sponsoring_org
So I will just remove it...we will have two test doing the same otherwise